Resources/views/Form/karser_recaptcha3_widget.html.twig

```diff
@@ -4,14 +4,17 @@
         {{ block('form_widget_simple') }}
 
         {% if form.vars.enabled -%}
-            <script src="https://www.google.com/recaptcha/api.js?render={{ form.vars.site_key }}"></script>
-            <script>
-                grecaptcha.ready(function() {
-                    grecaptcha.execute('{{ form.vars.site_key }}', {action: '{{ form.vars.action_name }}'}).then(function(token) {
-                        document.getElementById('{{ id }}').value = token;
+            <script type="text/javascript">
+                var recaptchaCallback = function() {
```

Comment:
The function could be overridden If multiple elements are used on a page

Reply:
This callback script should be printed only once per page, so it shouldn't be in this file (probably new service is required). I have no time to look deeper into your bundle so I will leave this tweak in not-so-pretty way in hope you will change it in the future ;)